server/src/main/kotlin/io/provenance/invoice/services/EventHandlerService.kt

```diff
@@ -43,12 +44,10 @@ class EventHandlerService(
             // Payments should only be allowed for oracle-approved invoices
             InvoiceStatus.APPROVED,
         )
-
-        private val HANDLER_THREAD_POOL = Executors.newFixedThreadPool(10)
```

Comment:
This was used for a previous implementation and does not any longer do anything at all